examples/validation-app/src/providers/my-validation-error-provider.ts

```diff
@@ -0,0 +1,31 @@
+import {HandlerContext, HttpErrors, RejectProvider} from '@loopback/rest';
```

Comment:
Please add copyright to the top of the file

Reply:
ran `slt copyright` in this example.